metaci/build/migrations/0019_build_planrepo.py

```diff
@@ -8,7 +8,11 @@
 def set_planrepository(apps, schema_editor):
     Build = apps.get_model('build', 'Build')
     for build in Build.objects.all().iterator():
-        build.planrepo = build.plan.planrepository_set.get(repo = build.repo)
+        try:
+            build.planrepo = build.plan.planrepository_set.get(repo = build.repo)
+        except Exception as e:
```

Comment:
you should be able to catch ObjectDoesNotExist instead, which is the base of all Model.DoesNotExist exceptions.